test/integration/command/config_test.py

```diff
@@ -89,10 +93,10 @@ def test_config_install_conanignore():
     })
 
     def _assert_config_exists(path):
-        assert os.path.exists(os.path.join(tc.cache_folder, path))
+        assert os.path.exists(os.path.join(tc.cache_folder, path)), f"{path} must exist"
```

Comment:
I think that assert messages in tests in many cases provide little value, but add some visual noise.